firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -451,4 +470,34 @@ private Void deleteFirebaseInstallationId() throws FirebaseInstallationsExceptio
     persistedInstallation.insertOrUpdatePersistedInstallationEntry(entry.withNoGeneratedFid());
     return null;
   }
+
+  private static boolean isIIDVersionCompatible() {
+    try {
+      final Class<FirebaseInstanceId> firebaseInstanceIdClass = FirebaseInstanceId.class;
+      String iidVersion = getIIDVersion();
+      if (TextUtils.isEmpty(iidVersion)
+          || Utils.compareVersion(IID_SDK_VERSION_BEFORE_FIS, iidVersion) >= 0) {
+        return false;
+      }
+    } catch (NoClassDefFoundError exception) {
+      // There is no IID SDK at all.
+      return true;
```

Comment:
code duplication of `return true`

Reply:
Updated the code better readability. Please review again.